src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
+$cell-size:100px;
+$gap-size:1px;
+$calendar-pading:10px;
+$width-calendar: ($cell-size*7)+($gap-size*6)+($calendar-pading*2);
+
+$week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5
+  "sun" 6;
+
+.container {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  width: $width-calendar;
+  flex-wrap: wrap;
+  margin: 20px auto;
+  padding: 10px;
+  gap: 1px;
+
+  &__day {
+    box-sizing: border-box;
+    width: 100px;
+    height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    display: flex;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+      transform: translateY(-20px);
```

Comment:
Let's add some transition for background-color and transform properties, so hover effect looks smoother)